### ISSUE TITLE
Pull fw project Id from xml

### ISSUE THREAD

**Issue by hahn-kev** [sillsdev/languageforge-lexbox#955]
this will allow us to determine duplicate projects and hopefully clean them up.

**Comment by rmunn** [comment]
First few lines of the `General/LanguageProject.langproj` file should be:

```xml
<?xml version="1.0" encoding="utf-8"?>
<LanguageProject>
        <LangProject
                guid="e2472f2d-4a99-4e97-972a-898f411b0a57">
```

So we don't really need to parse the whole XML file to get this; we just need to run `hg cat`, perhaps piped through `head` for further efficiency (so that once it gets past the first ten lines it will close the pipe which will in turn cause the `hg cat` process to end). Then grep for `<LangProject` and grab the first line that contains `guid="..."` on or after the `<LangProject` line. That way whether the format stays the same (each attribute on one line) or whether a future version of Chorus changes it to `<LangProject guid="...">` on a single line, the command-runner command will continue to work.

**Comment by rmunn** [comment]
Decided to tackle this as part of #952 (the PR that fixes #922) rather than make a separate PR, since this and #922 share quite a lot of concepts.
